packages/ember-input-validation/src/components/tpk-validation-input.gts

```diff
@@ -115,11 +117,8 @@ export default class TpkValidationInputComponent extends BaseValidationComponent
           maxlength={{@maxlength}}
           data-test-input-not-yielded
         />
-        {{yield
-          (hash
-            errors=this.errors hasError=this.hasError firstError=this.firstError
-          )
-        }}
+        <TpkValidationErrorsComponent @errors={{this.errors}} @classless={{@classless}} />
+        {{yield (hash hasError=this.hasError firstError=this.firstError)}}
```

Comment:
C'est doublon d'avoir le yield et le error component, mais en vrai, je suggère de juste le retirer ici (le <TpkValidationErrorsComponent @errors={{this.errors}} @classless={{@classless}} /> )

Et on le remettra dans le prefab d'input "input"